test_bms.cpp

```diff
@@ -0,0 +1,25 @@
+#define CATCH_CONFIG_MAIN  // This tells Catch to provide a main() - only do this in one cpp file
+
+#include "bms.h"
+#include "test/catch.hpp"
+
+/*
+Input ouput
+{3}
+{3,3}

```

Comment:
this should be 3-3, 2 if we follow the next line 3,3,4 => 3-4,3
If 3,3 is empty, then a test case should be present for such exceptions.

Reply:
are the test cases updated for test data mentioned in the comments?
Let me know if it is done....